### ISSUE TITLE
Bounding box size is off

### ISSUE THREAD

**Issue by stalagmite7** [WongKinYiu/yolor#279]
I am trying to train a yolor-p6 model on my own custom dataset (paper branch). Using default data preprocessing of "mosaic" (not feeding in the `--rect` flag while starting the training script)

The box in the left image is the ground truth, the box in the right image is the prediction (for the same class)
<img width="1536" alt="image" src="https://user-images.githubusercontent.com/12752337/194115402-e919e9af-018d-406f-9969-89fca7235187.png">

Not sure why this is happening since I didn't modify anything in the code; plus it worked well when I tested the yolor-p6 model out of the box on the included horses image (boxes were correct/tightly fitted around the horses)

**Comment by stalagmite7** [comment]
My mistake - the data I was feeding in was in normalized xyxy format, whereas it expected it in normalized xywh format. Changed whatever was necessary and works with my data now.
